### PR TITLE
Redux setup

### PR DESCRIPTION
**Pull request by tikagan**
closes issue #2

### CODE REVIEW

**Comment by tikagan** [comment]
running tests in the CLI says that they've all passed but I'm not sure what the issue is that is making them fail here :/

File: src/actions/boycottActions.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// export const POKEMON_ACTIONS = {
+//     SET_IS_LOADING: 'pokemon/SET_IS_LOADING',
+//     SET_POKEMON: 'pokemon/SET_POKEMON',
+//   };
+
+export const BOYCOTT_ACTIONS ={
+    
+}
```
Comment (jesselizdavid) [inline src/actions/boycottActions.js:8]: EOF :)

**Review by jesselizdavid — COMMENTED** [review]
I wish I could be more helpful on reviewing this PR but knowledge of react-redux is limited.  @meaghaneleanor you might be able to give more useful comments?

**Comment by meaghaneleanor** [comment]
@jesselizdavid @tikagan i think the most important is that the redux package is installed/initialized, otherwise everything else looks good!

**Review by meaghaneleanor — DISMISSED** [review]
lookin good!

File: src/store/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import { rootReducer } from '../reducers/rootReducer';
+import thunk from 'redux-thunk';
+
+export default function configureStore() {
+  return createStore(
+    rootReducer,
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
+    applyMiddleware(thunk)
+  );
+}
```
Comment (meaghaneleanor) [inline src/store/configureStore.js:11]: EOF :)

File: src/reducers/rootReducer.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+//import { pokemonReducer } from './pokemon.reducer';
+
+
+export const rootReducer = combineReducers({
+    //pokemon: pokemonReducer,
+  });
```
Comment (meaghaneleanor) [inline src/reducers/rootReducer.js:7]: EOF :)

**Review by meaghaneleanor — APPROVED** [review]

**Comment by meaghaneleanor** [comment]
Just added a few comments about adding a new EOF line, but looks good otherwise 👍

**Pull request merged**